.circleci/local_publish_helpers_codebuild.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+git checkout $BRANCH_NAME
+
```

Comment:
We shouldn't checkout a branch every time we load this script.
Is this intentional?